lang/pypy/Portfile

```diff
@@ -176,15 +186,26 @@ if {${python.branch} == 2.7} {
     patchfiles-append   pypy3-darwin.py.diff
 
     checksums           rmd160  a977b07ac5ae199054563eb0cfbb9f5ed8acd11d \
-                        sha256  2abaa1e9fe1ec0e233c9fbc377a0c8e9a0634080a8f4f30eb6898301f6618c12 \
-                        size    26976726
-
+      sha256  a77340a91e81aedf02e78e6e2ea96806f76b50256c6338067f5ff540edbae072 \
+                          size    28200000
     set use_prefix yes
-}
+} elseif {${python.branch} == 3.10} {
```

Comment:
This too is a possibly undesired numeric comparison. `3.10` will be equal to `3.1` and `3.100`.

Reply:
eq "3.10" ?